plugins/opusCodec/src/OpusDecoder.h

```diff
@@ -0,0 +1,29 @@
+#ifndef OPUSDECODER_H
+#define OPUSDECODER_H
+
+
+#include <plugins/CodecPlugin.h>
+#include "opus/opus.h"
```

Comment:
Since this is an external library header please include it as <opus/opus.h> instead of using the local include style with double quotes